src/ext/oblivc/obliv_bits.h

```diff
@@ -40,13 +43,17 @@ void __obliv_c__setBitOr(OblivBit* dest,const OblivBit* a,const OblivBit* b);
 void __obliv_c__setBitXor(OblivBit* dest,const OblivBit* a,const OblivBit* b);
 void __obliv_c__setBitNot(OblivBit* dest,const OblivBit* a);
 void __obliv_c__flipBit(OblivBit* dest); // Avoids a struct copy
+void __obliv_c__floatAdd(OblivBit* dest,const OblivBit* a,const OblivBit* b);
```

Comment:
The ones above it yes, but elsewhere we have at least these three styles:
floatAdd
setFloatKnown
setPlainAddF

Unless I am mistaken, all of these represent circuits over many bits. Is there a reason for them to be differently named (an honest question - maybe there is)?